### PR TITLE
add goosefx to solana dex

### PR DESCRIPTION
**Pull request by andrewhong5297**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by andrewhong5297** [comment]
I've checked the values here, and asked the goosefx team to confirm accuracy. Data looks good 👍 

https://dune.com/queries/3205576/5358346

**Comment by Hosuke** [comment]
By the way, @jeff-dude do we need to add `dex_solana.trades` into `dex.trades_beta`? 
Or we can keep it independent for now.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @andrewhong5297 

[Sample goosefx_ssl_v2_solana.trades](https://dune.com/queries/3221858)

File: models/goosefx/solana/goosefx_ssl_v2_solana_trades.sql [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        FROM {{ source('goosefx_solana', 'gfx_ssl_v2_call_createPair') }} ip 
+        LEFT JOIN {{ ref('tokens_solana_fungible') }}  tkA ON tkA.token_mint_address = ip.account_mintOne
+        LEFT JOIN {{ ref('tokens_solana_fungible') }}  tkB ON tkB.token_mint_address = ip.account_mintTwo
+    )
+
+    , all_swaps as (
+        SELECT 
+            sp.call_block_time as block_time
+            , 'goosefx_ssl' as project
+            , 2 as version
```
Comment (Hosuke) [inline models/goosefx/solana/goosefx_ssl_v2_solana_trades.sql:41]: We may use `'2'` as varchar typed version.
```suggestion
            , '2' as version
```

Comment (andrewhong5297) [reply models/goosefx/solana/goosefx_ssl_v2_solana_trades.sql:41]: I used integers for all versions across dex_solana trades for now, so we can't do this

**Comment by jeff-dude** [comment]
> By the way, @jeff-dude do we need to add `dex_solana.trades` into `dex.trades_beta`? Or we can keep it independent for now.

we're keeping independent for now. @andrewhong5297 has agreed to have separate and then help refactor the design a little later on. i'd recommend soon-ish to avoid too many models to have to do 🥲

**Review by jeff-dude — APPROVED** [review]

**Comment by andrewhong5297** [comment]
> > By the way, @jeff-dude do we need to add `dex_solana.trades` into `dex.trades_beta`? Or we can keep it independent for now.
> 
> we're keeping independent for now. @andrewhong5297 has agreed to have separate and then help refactor the design a little later on. i'd recommend soon-ish to avoid too many models to have to do 🥲

haha just one more dex, and I'm waiting for a new token standard to get decoded on Solana. Then I promise to do the rework 😁

**Pull request merged**